pkgs/tools/misc/hyfetch/default.nix

```diff
@@ -2,13 +2,13 @@
 
 buildPythonPackage rec {
   pname = "HyFetch";
-  version = "1.3.0";
+  version = "1.4.0";
 
   src = fetchFromGitHub {
     repo = "hyfetch";
     owner = "hykilpikonna";
-    rev = version;
-    sha256 = "sha256-8Mp3MV9HVzXzT/W6F/lD34tT0uOgqyydg31PlR3sMUA=";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-15FeW3yaM7eUJ1O20H/bQquDRAsRWTJMzkzQ5Kv1ezE=";
   };
 
   # TODO: Remove with next release bump since it has been fixed upstream (hykilpikonna/hyfetch@d797a8c)
```

Comment:
This and the following 4 lines can be removed with this version bump now (not sure how to handle that with r-ryantm though, aside from opening a new PR or having a nixpkgs committer do it)